packages/pilot/src/components/RiskLevel/style.css

```
@@ -46,3 +46,45 @@
   background-color: #fd003f;
   width: 100%;
 }
+
+@media print {
+
+  .riskLevelBar {
+    display: inline-block;
+    width: 100%;
+    position: relative;
+    border-top: 10px solid var(--color-light-silver-50);
+  }
+
+  .riskLevel {
+    height: 10px;
+    position: absolute;
+    top: -10px;
+    left: 0;
+  }
+
+  .very_low {
+    border-top: 10px solid #04e12d;
```

Poderia rolar umas variáveis para estas cores ex: 
```suggestion
:root {
   --rist-very-low: #04e12d;
   ...
 }
 ...
    border-top: 10px solid var(--rist-very-low);
```